chaoshub/cli.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import io
+import json
+from logzero import logger
```

Comment:
This one should go after the `import click` statement as it's a third-party lib :)

Reply:
Done.